change/@internal-react-components-b3c4d654-0777-4323-9b82-82a374362193.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Fix teams message strange margin",
```

Comment:
Might I suggest a more descriptive changelog.
`Bugfix for messages from teams users having extra margins around the message content`